litestar/_openapi/responses.py

```diff
@@ -204,9 +207,12 @@ def create_error_responses(exceptions: list[type[HTTPException]]) -> Iterator[tu
             schema = Schema(one_of=exceptions_schemas)
         else:
             schema = exceptions_schemas[0]
+
+        _schema = schema_creator.process_schema_result(schema)
```

Comment:
Y the underscore?

Reply:
mypy/pyright complaining i think, i'll check again.